unlock-protocol-com/src/components/layout/DefaultLayout.tsx

```diff
@@ -132,12 +132,24 @@ const MENU = {
           description: 'Deploy your membership contract with few clicks.',
           url: `${unlockConfig.appURL}/locks`,
         },
+        {
+          title: 'ALPHAtweet',
+          src: '/images/marketing/img-aphatweet.png',
+          url: 'https://alphatweet.xyz/',
+          description: 'Micro-blogging with a twist of referral.',
```

Comment:
@julien51 is this right description?

Reply:
That's for @ccarfi  our wording wizzard!